CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.5] - 2024-10-15
```

Comment:
It's prob okay to have exchange bump in this PR as well since you're including these in the changelog (and saves a pr)

Reply:
sgtm! i was debating that - forgot how we wanted to rev those packages

Reply:
35a5371